src/lib/ngsi/ParseData.h

```diff
@@ -253,14 +263,15 @@ typedef struct UpdateContextData
 *
 * UpdateContextResponseData - 
 */
-typedef struct UpdateContextResponseData
+struct UpdateContextResponseData
 {
+  UpdateContextResponseData(): cerP(0), attributeP(0), metadataP(0), domainMetadataP(0) {}
```

Comment:
I do prefer NULL, no make clear that the attribute referes to a point instead that to an `int`. However, not sure if NULL can be used in the initialization list...